demos/place_recognition_demo/python/place_recognition_demo/place_recognition.py

```diff
@@ -31,24 +31,24 @@ class IEModel: # pylint: disable=too-few-public-methods
     def __init__(self, model_path, device, cpu_extension):
         log.info('OpenVINO Inference Engine')
         log.info('\tbuild: {}'.format(get_version()))
-        ie = IECore()
+        core = Core()
         if cpu_extension and device == 'CPU':
-            ie.add_extension(cpu_extension, 'CPU')
+            core.add_extension(cpu_extension, 'CPU')
 
         log.info('Reading model {}'.format(model_path))
-        self.net = ie.read_network(model_path, model_path.with_suffix('.bin'))
-        self.input_name = next(iter(self.net.input_info))
-        self.output_name = next(iter(self.net.outputs))
-        self.input_size = self.net.input_info[self.input_name].input_data.shape
-        self.exec_net = ie.load_network(network=self.net, device_name=device)
+        self.model = core.read_model(model_path, model_path.with_suffix('.bin'))
```

Comment:
can we unify model reading (in some demos, only model provided, in some demos 2 files using with_suffix, below also model_path[:-4] + '.xml'/',bin' used?

Reply:
Yeah, I will also unify it in the other PR once the all py demos will be migrated to ov2.0 and merged. Are you okay with it?

Reply:
yes, it is not stopper, just common recommendation :)